docs/resources/cluster.md

```diff
@@ -76,6 +86,8 @@ resource "cockroach_cluster" "basic" {
 
 ### Optional
 
+- `backup_config` (Attributes) The backup settings for a cluster.
+ Each cluster has backup settings that determine if backups are enabled, how frequently they are taken, and how long they are retained for. Use this attribute to manage those settings. (see [below for nested schema](#nestedatt--backup_config))
```

Comment:
```suggestion
 Each cluster has backup settings that determine if backups are enabled, how frequently they are taken, and how long they are retained for. Use this attribute to manage those settings. Refer to [Nested Schema for `backup_config`](#nestedatt--backup_config).
```
Like the other instance, wondering if you could just use the section header for Nester Schemas here, instead of the anchor link.